modules/subdyn/src/SubDyn.f90

```diff
@@ -1166,8 +1166,8 @@ SUBROUTINE SD_Input(SDInputFile, Init, p, ErrStat,ErrMsg)
    deallocate(StrArray)
    CALL AllocAry(StrArray, nColumns, 'StrArray',ErrStat2,ErrMsg2); if (Failed()) return 
    CALL ReadCAryFromStr ( Line, StrArray, nColumns, 'Members', 'First line of members array', ErrStat2, ErrMsg2 ); if(Failed()) return
-   call LegacyWarning('Member table contains 6 columns instead of 7,  using default member directional cosines ID (-1) for all members.\
-   The directional cosines will be computed based on the member nodes for all members.')
+   call LegacyWarning('Member table contains 6 columns instead of 7,  using default member directional cosines ID (-1) for all members. &
+   &The directional cosines will be computed based on the member nodes for all members.')
```

Comment:
I didn't know you could continue a string to the next line this way.

Reply:
I don't typically use this format for continuing a string, but it's legal: https://www.intel.com/content/www/us/en/docs/fortran-compiler/developer-guide-reference/2023-1/free-source-form.html

Reply:
I learned something new today :)